WangscapeTest/TestReseedable.cpp

```diff
@@ -3,7 +3,8 @@
 #include <noise/MakeReseedable.h>
 #include <noise/module/ModuleFactories.h>
 
-class TestReseedable : public ::testing::Test {
+class TestReseedable : public ::testing::Test
```

Comment:
This is a bug in libnoise. The documentation for `noise::module::Clamp` ([link](http://libnoise.sourceforge.net/docs/classnoise_1_1module_1_1Clamp.html)) clearly states "The lower bound must be less than or equal to the upper bound.", but the assertion checks `<` rather than `<=`.

This is present in the official source on sourceforge, so I'm not sure why this code was working on my machine. Wangscape will need to support this use case, so I'll replace any usage of `Clamp` modules with a pair of `Min` and `Max` modules.